packages/fields/src/components/create-template-part-modal/index.tsx

```diff
@@ -152,10 +208,14 @@ export function CreateTemplatePartModalContents( {
 						label={ __( 'Area' ) }
 						className="editor-create-template-part-modal__area-radio-group"
```

Comment:
Should I update these classes even if it is a temporary solution=

Reply:
Yeah I think we should update them ideally `fields` instead of `editor` basically.

Reply:
I updated it with 199c313b51f724a6f26bfc186e3a0e579e369d27